src/Worker/Core/IExceptionPropertiesProvider.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Question: is this the right place to put this file? Customer need to refer this interface so they need to use this package. I guess I should put it here but I am not 100% sure.